### PR TITLE
Add poll command

### PR DESCRIPTION
**Pull request by natwhite**

### CODE REVIEW

File: src/bot/commands/utilities/poll.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import {Command, Input} from '@api';
+import {TextChannel} from 'discord.js';
+import {UserPoll} from "@libraries/utilities/create_poll";
+import {Emoji} from "@libraries/emoji";
+
+export class Poll extends Command {
+    constructor() {
+        super({
+            name: 'poll',
```
Comment (baileyherbert) [inline src/bot/commands/utilities/poll.ts:9]:
```suggestion
            name: 'poll',
            aliases: ['vote'],
```

File: src/bot/commands/utilities/poll.ts [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+            description: 'Lets members vote on between 2-6 options',
+            arguments: [
+                {
+                    name: 'question',
```
Comment (baileyherbert) [inline src/bot/commands/utilities/poll.ts:14]:
```suggestion
                    name: 'question',
                    description: 'The question to start a poll for, in quotation marks.',
```

Help information looks gross without argument descriptions.

File: src/bot/commands/utilities/poll.ts [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+                    constraint: 'string',
+                    required: true,
+                },
+                {
+                    name: 'choices',
```
Comment (baileyherbert) [inline src/bot/commands/utilities/poll.ts:20]:
```suggestion
                    name: 'choices',
                    description: 'A list of choices separated by commas.',
```

And this really needs to be here.

**Review by baileyherbert — APPROVED** [review]

File: src/bot/commands/utilities/poll.ts (hunk 2 above)
Comment (natwhite) [reply src/bot/commands/utilities/poll.ts:14]: Of course. I forgot the framework handled that for multiple parameters.

**Pull request merged**